tests/components/androidtv/test_media_player.py

```diff
@@ -146,15 +138,12 @@ def _setup(hass, config):
         assert state.state == STATE_IDLE
 
     if patch_key == "python":
-        assert (
-            "ADB connection to 127.0.0.1:5555 successfully established"
-            in caplog.record_tuples[2]
-        )
+        assert ("ADB connection to 127.0.0.1:5555 successfully established" in
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4678217)